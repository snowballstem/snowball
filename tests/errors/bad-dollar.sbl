strings ( x )

externals ( stem )

booleans ( b )

groupings ( g )

define stem as (
    $ = 'xxx'
    $stem = 1
    $b = 1
    $g = 1
)
