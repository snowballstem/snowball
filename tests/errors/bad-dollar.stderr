errors/bad-dollar.sbl:10: unexpected = in integer test expression after $
errors/bad-dollar.sbl:11: 'stem' not of type integer or string
errors/bad-dollar.sbl:12: 'b' not of type integer or string
errors/bad-dollar.sbl:13: 'g' not of type integer or string
